pkg/response/response.go

```diff
@@ -82,3 +82,27 @@ func WriteError(request *restful.Request, response *restful.Response, httpStatus
 		fmt.Sprintf("error: %v: %v", errorResponse, eventErr))
 	fmt.Printf("%+v\n", eventErr)
 }
+
+// WriteErrorWithEventID sends error message with Event ID
+func WriteErrorWithEventID(request *restful.Request, response *restful.Response, httpStatusCode int,
```

Comment:
can we combine some of the function body with `WriteError` so that we don't duplicate the code too much?

Reply:
I imagine we can make `WriteError` is just calling `WriteErrorWithEventID` but the event ID is `errorResponse.errorCode`

Reply:
Yes now updated. sorry for late update.